authors/settings.py

```diff
@@ -81,8 +81,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'ah-scorpion',
+        'USER': 'team-scorpion',
+        'PASSWORD': 'password',
+        'HOST': 'localhost',
+        'PORT': '',
```

Comment:
I would like you to implement this with an abstraction that hides sensitive information like `NAME`,  `USER`, `PASSWORD`,  `LOCALHOST` and `POST` in github. Exposing that information is a security lapse.
Check this implementation here on how to add a .env file with such confidential information. https://gist.github.com/bennylope/2999704